Move prover returns: exiting with verification errors
error: abort code not covered by any of the `aborts_if` or `aborts_with` clauses
   ┌─ tests/sources/functional/aborts_if_with_code.move:38:5
   │
31 │               abort 2
   │               ------- abort happened here with code 0x2
   ·
38 │ ╭     spec conditional_abort_invalid {
39 │ │         aborts_if x == 1 with 1; // wrong code
40 │ │         aborts_if y == 2 with 3;
41 │ │         ensures result == x;
42 │ │     }
   │ ╰─────^
   │
   =     at tests/sources/functional/aborts_if_with_code.move:29: conditional_abort_invalid
   =         x = <redacted>
   =         y = <redacted>
   =     at tests/sources/functional/aborts_if_with_code.move:30: conditional_abort_invalid
   =     at tests/sources/functional/aborts_if_with_code.move:31: conditional_abort_invalid
   =         ABORTED

error: abort code not covered by any of the `aborts_if` or `aborts_with` clauses
   ┌─ tests/sources/functional/aborts_if_with_code.move:48:5
   │
46 │           10 / x
   │           ------ abort happened here with execution failure
47 │       }
48 │ ╭     spec exec_failure_invalid {
49 │ │         aborts_if x == 0 with 1; // wrong code
50 │ │         ensures result == 10 / x;
51 │ │     }
   │ ╰─────^
   │
   =     at tests/sources/functional/aborts_if_with_code.move:45: exec_failure_invalid
   =         x = <redacted>
   =     at tests/sources/functional/aborts_if_with_code.move:46: exec_failure_invalid
   =         ABORTED

error: abort code not covered by any of the `aborts_if` or `aborts_with` clauses
   ┌─ tests/sources/functional/aborts_if_with_code.move:77:5
   │
74 │               abort(2)
   │               -------- abort happened here with code 0x2
   ·
77 │ ╭     spec aborts_if_with_code_mixed_invalid {
78 │ │         aborts_if x == 1;
79 │ │         aborts_if x == 2 with 1;
80 │ │     }
   │ ╰─────^
   │
   =     at tests/sources/functional/aborts_if_with_code.move:69: aborts_if_with_code_mixed_invalid
   =         x = <redacted>
   =     at tests/sources/functional/aborts_if_with_code.move:70: aborts_if_with_code_mixed_invalid
   =     at tests/sources/functional/aborts_if_with_code.move:73: aborts_if_with_code_mixed_invalid
   =     at tests/sources/functional/aborts_if_with_code.move:74: aborts_if_with_code_mixed_invalid
   =         ABORTED

error: abort code not covered by any of the `aborts_if` or `aborts_with` clauses
    ┌─ tests/sources/functional/aborts_if_with_code.move:105:5
    │
102 │               abort(2)
    │               -------- abort happened here with code 0x2
    ·
105 │ ╭     spec aborts_with_invalid {
106 │ │         aborts_with 1,3;
107 │ │     }
    │ ╰─────^
    │
    =     at tests/sources/functional/aborts_if_with_code.move:97: aborts_with_invalid
    =         x = <redacted>
    =     at tests/sources/functional/aborts_if_with_code.move:98: aborts_with_invalid
    =     at tests/sources/functional/aborts_if_with_code.move:101: aborts_with_invalid
    =     at tests/sources/functional/aborts_if_with_code.move:102: aborts_with_invalid
    =         ABORTED

error: abort code not covered by any of the `aborts_if` or `aborts_with` clauses
    ┌─ tests/sources/functional/aborts_if_with_code.move:131:5
    │
128 │               abort(1)
    │               -------- abort happened here with code 0x1
    ·
131 │ ╭     spec aborts_with_mixed_invalid {
132 │ │         pragma aborts_if_is_partial = true;
133 │ │         aborts_if x == 1 with 1;
134 │ │         aborts_with 2;
135 │ │     }
    │ ╰─────^
    │
    =     at tests/sources/functional/aborts_if_with_code.move:123: aborts_with_mixed_invalid
    =         x = <redacted>
    =     at tests/sources/functional/aborts_if_with_code.move:124: aborts_with_mixed_invalid
    =     at tests/sources/functional/aborts_if_with_code.move:127: aborts_with_mixed_invalid
    =     at tests/sources/functional/aborts_if_with_code.move:128: aborts_with_mixed_invalid
    =         ABORTED
